.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: generic
 sudo: required
-install:
-  - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
 script: swift test
 matrix:
   include:
     - os: osx
-      osx_image: xcode8.3
+      osx_image: xcode9.4
     - os: linux
       dist: trusty
-      before_install:
-        - sudo hostname travis-ci.local
```

Comment:
Why remove this? The hostname should end with `.local`, see also the travis build failing because of this.

Reply:
I should not have been deleted ) Is deleted when you commit "Build against 4.1 on Travis"